R/taudem-exec.R

```diff
@@ -31,7 +32,8 @@
 #'     "-fel", file.path(test_dir,"filled_pits.tif")
 #'   )
 #' )
-taudem_exec <- function(program, args, n_processes = getOption("traudem.n_processes", 1), quiet = getOption("traudem.quiet", FALSE)) {
+taudem_exec <- function(..., program, args, n_processes = getOption("traudem.n_processes", 1), quiet = getOption("traudem.quiet", FALSE)) {
```

Comment:
I thought the signature would be `program, args, ..., n_processes, quiet` . It's fine to have the first few arguments before the ellipsis.